doc/specs/friends.md

```diff
@@ -0,0 +1,67 @@
+# Authentication system
```

Comment:
Wrong title. 

Template needs to be updated to remove author and changelog, redundant considering github...